mkosi/config.py

```diff
@@ -994,6 +995,12 @@ class MkosiConfigParser:
             parse=config_parse_feature,
             help="Use btrfs subvolumes for faster directory operations where possible",
         ),
+         MkosiConfigSetting(
+            dest="seed",
```

Comment:
Please also set metavar

Reply:
Would this be a proper metavar? `UUID|random`

Reply:
I would just use UUID